File: Iniciante/1012.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double A, B, C, pi = 3.14159;

    cin >> A >> B >> C;

    cout << fixed << setprecision(3);
    cout << "TRIANGULO: " << (A*C)/2 << endl;
    cout << "CIRCULO: " << pi*C*C << endl;
    cout << "TRAPEZIO: " << (C*(A+B))/2 << endl;
    cout << "QUADRADO: " << B*B << endl;
    cout << "RETANGULO: " << A*B << endl;

    return 0;
}

File: Beginner/1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(/\s+/);

let pi = 3.14159

let A = Number(lines[0])
let B = Number(lines[1])
let C = Number(lines[2])

let areaTri = (A * C) / 2
let areaCir = pi * (C * C)
let areaTra = ((A + B) * C) / 2
let areaQua = B * B
let areaRec = A * B

console.log("TRIANGULO: " + areaTri.toFixed(3))
console.log("CIRCULO: " + areaCir.toFixed(3))
console.log("TRAPEZIO: " + areaTra.toFixed(3))
console.log("QUADRADO: " + areaQua.toFixed(3))
console.log("RETANGULO: " + areaRec.toFixed(3))